src/recipes.ts

```diff
@@ -33,12 +33,12 @@ export const createRecipesFromLocalStorage = () => {
         return
     }
 
-    const recipeIndex = 1
-    const receivedRecipes = Storage.get('recipes')
+    const recipeIndex: number = 1
+    const receivedRecipes: Array<Object> = Storage.get('recipes')
```

Comment:
you can create generic get from storage 😉 :
```
Storage.get<Array<Recipe>>('recipes')
```

inside get you need to create generic function and cast response to T